actionpack/test/dispatch/request_test.rb

```diff
@@ -1061,7 +1061,7 @@ class RequestParameterFilter < BaseRequestTest
 
       filter_words << 'blah'
       filter_words << lambda { |key, value|
-        value.reverse! if key =~ /bargain/
```

Comment:
(If "this change" means change from `value.reverse! if key =~ /bargain/` to `key =~ /bargain/ ? value.reverse : value`,) after this commit procs given to `filter_parameters` are expected to return filtered `value` (if filtering is needed) instead of destruct passed `value`.
Although `String#reverse!` returns reversed string, I think it is better to use "String#reverse" because we do only need to **return** filtered `value`.